inst/doc/geoknife.R

```diff
@@ -4,6 +4,9 @@ options(continue=" ")
 options(width=60)
 library(knitr)
 library(geoknife)
+query <- geoknife::query
+`values<-` <- geoknife::`values<-`
+id <- geoknife::id
```

Comment:
or I could just do the `::` inline w/ the calls, I just thought that was messy. Odd thing is that you don't need to do this for this vignette, and the other vignette (that causes the collision) doesn't used these potentially masked functions. I think this is maybe something devtools shouldn't do - whatever is happening in the background of build_vignettes - because these vignettes should effectively be separate for their evaluations...I'm on the fence.